scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 617,
-  MAX_SIZE: 916,
+  MAX_SIZE: 922,
```

Comment:
This is actually only an increase of 1, since the largest total bundle size is currently 916 on latest, and this PR makes that 917 (+ 5 = 922)